deploy_to_openshift.sh

```diff
@@ -184,6 +186,7 @@ then
   #default parameters
   stress_profile="very-light"
   evacuate_node="false"
+  stressor_image="docker.io/pranjalgupta2199/log-stressor:v1"
```

Comment:
@ajaygupta978  How will someone (which is not you) be able to use this, change this, or know what is included in this image. I think that this is not what we need! 

The entire idea was that this relaies on `registry.fedoraproject.org/fedora` so there is no "specific" code anywhere and that you can change the framework and see the changes deployed

Reply:
The only option I see to merge such a change is if you make this optional ... so that with some parameter you can override the default and use your own image ... but in any event not rely on your private image in this repo

Reply:
@Pranjal-Gupta2 if you can discuss this with @ajaygupta978 :-) thanks

Reply:
@eranra I have updated this PR to use [cluster-logging-load-client](https://github.com/ViaQ/cluster-logging-load-client) image. Can you please review it ?

Reply:
@Pranjal-Gupta2 I will delegate this to @ajaygupta978 if this is ok.

Reply:
@ajaygupta978 I do not want to postpone this. I fully trust your judgment on this PR. I will really appreciate If you will be able to work with @Pranjal-Gupta2 to finish the review and merge :-)  THX

Reply:
Thanks @eranra we have discussed and reviewed. @Pranjal-Gupta2 needs to update readme then I can merge it.